scripts/package_antares_web.sh

```diff
@@ -15,7 +15,7 @@ ANTARES_SOLVER_VERSION_INT="880"
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
 PROJECT_DIR=$(dirname -- "${SCRIPT_DIR}")
 DIST_DIR="${PROJECT_DIR}/dist/package"
-RESOURCES_DIR="${PROJECT_DIR}/resources"
+RESOURCES_DIR="${PROJECT_DIR}/resources/"
```

Comment:
Note that you will have a double slash later when you append a slash: it works but I think it's better to not have it.